### ISSUE TITLE
[Question]: Google Play Release

### ISSUE THREAD

**Issue by mitch133** [nova-video-player/aos-AVP#633]
### Description

Any idea when nova V6 will be available through the Google Play Store? I thought it would be a matter of days after reading comments weeks ago saying Nova was back on track after some issues with Google. But still waiting

**Comment by Hotboy1987** [comment]
me too want to know

**Comment by courville** [comment]
The current situation is the following: I had issues in the past with Google that was seeing the donation button but this was proven to be an error in their review process (and still is).
Now all my submissions to google play store need to undergo manual review. The last submission is still under review and has been for 3 weeks.
At this point we all just need to wait :-(

**Comment by courville** [comment]
Status update.
Google has two bugs in their review process and current level of support I am able to reach does not help me getting a resolution on the issues I face.
Current published release of nova (v5) displays a sponsor link when the apk is sideloaded but not installed from Google Play. The sponsor link is against Google Play policy.
The review process does not install the apk under review via Google Play but directly via adb (sideloaded) and thus any new submission using a sponsor link displayed only if not installed via Google Play would fail (first bug but hey this is acceptable).
The tricky part comes when submitting a new corrective apk as internal/beta release where this behavior is disabled.
In this submission Google unfortunately, the Google check of sponsor link presence is performed not on the new apk but the already released one (second bug). This results in an always failing submission (screenshots reported in the refusal email are the ones from v5 nova release version...).
As a consequence, I am not able to release any internal/alpha/beta releases.
My only way out is to go for a full release submission for nova v6 hoping that this will unlock the situation.
Note that even if I explained clearly the issue to Google, it seems that I am only able to reach automated AI level support without a real human brain to read and properly think about my peculiar report and status. 

Note that last submission remained under review for 5 weeks without any Google response (I had raise this non activity to the support team to progress and lead to a refusal again).

FYI, I just submitted the new v6 of nova as full release deployed at 5% for now pending Google review.
